### PR TITLE
Labelling policy

### PR DESCRIPTION
**Pull request by BenJam**
Here's a first go at a labelling policy. I've tried to be really explicit about the size of an issue versus the skills needed to tackle it versus the stage of the process it might be in. 

I'm specifically trying to make it easier for contributors who are not developers to take part. So labels that these contributors can pick up immediately would be really helpful.

There are a couple of modifications to existing tickets and a certain degree of relabelling that might need to be done, which I'm happy to put my hand up for. 

I have proofed and spell checked this time!

### CODE REVIEW

File: labelling.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## Labelling 
+Labelling is a key part of our workflow. We label issues by size, type or skills needed and drive parts of our process using them. Specificly we label tickets that are ideal for first time contributors or those that are part of our funded commitments. 
+
+We will enforce this policy to ensure that new contributors and existing contributors alike are as productive and efficient with their time as possible. This may include modifying labels or rejecting a ticket if is does not comply with the following:
```
Comment (andrew) [inline labelling.md]: > We will enforce this policy

Maybe could be written in a less combative, more friendly way?

File: labelling.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Type or Skills
+Projects typically indicate the type of issue, but we would like to be more explicit. We would like these labels to reflect the skills needed of the person who would ideally tackle it. This will lead to more effective prioritisation and enable contriutors who wish to learn these skills more scope to do so. Feel free to label a ticket with a number of skills needed. If a ticket requires more tha two skills consider splitting into two or more tickets. 
+
+* `API` are issues regarding our own API. 
+* `content` these isssues require a little word-smithing of new or existing copy. This might be grammar, phrasing, voicing or similar types of issue.
+* 'design' is for pure design. This might involde the creation of assets like logos, fonts, illustrations or it may be in support of frontend or UX issues.
```
Comment (andrew) [inline labelling.md]: missing backticks here

File: labelling.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* `dependency parsing` in order to understand the relationships between libraries, their constituent components, and the projects they're included within, we have to parse dependency manifests. Sometimes these manifests require compilation, typically in the language or framework itself.
+* `documentation` If an issue requires changes or clarification to our documentation (READMEs, wiki, code comments, etc.) then it will be marked with this label. These issues normally do not pertain to code changes and are ideal for those who don't necessarily want to get their hands dirty with code.
+* `frontend` This relates to visual elements of the site that could be improved or are broken. This may be as simple as a change to a CSS colour or font size but may stretch all the way to things that do not render as expected on mobile devices. These are usually good issues for people with front-end development experience to tackle.
+* 'registry' Libraries.io gets it's data by scraping registries like package managers. Support for new registries or issues with current registries will be tagged with this label.
```
Comment (andrew) [inline labelling.md]: backticks

File: labelling.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* `frontend` This relates to visual elements of the site that could be improved or are broken. This may be as simple as a change to a CSS colour or font size but may stretch all the way to things that do not render as expected on mobile devices. These are usually good issues for people with front-end development experience to tackle.
+* 'registry' Libraries.io gets it's data by scraping registries like package managers. Support for new registries or issues with current registries will be tagged with this label. 
+* `research` in order to progress an issue or idea we may need more information about our own site, our users or more context regarding a problem for other sources. Any issue that needs a reasonable amount of information gathering will be tagged research. 
+* `security` If you spot an issue related to security issues (e.g. invalid SSL certs, potential CSRF issues), mark it with this label. Don't forget to be a good OSS citizen and always report zero-day issues through a private channel to minimise impact to your fellow users!
```
Comment (andrew) [inline labelling.md]: Perhaps we should setup a security mailing list? security@libraries.io

File: labelling.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+* `refactoring` If an issue requires code to be refactored before a particular change can be made, or if you spot inelegant patterns or implementations in code that you feel could be better, then feel free to add this label to an issue. If the issue is the latter type, please be careful about the language you use in these threads. For example, things like "this code sucks!" or "you must be an idiot!" are unacceptable!. Programmers have feelings too and there are ways to suggest code changes without insulting people!
+*`wontfix` This means that, after considering your issue in full, your issue is outside of [the intended scope of the project](/strategy.md) and is not something we'd like to add to the codebase right now or in the future. These are used sparingly and are intended to be rare, and are never used without reasoned justification.
```
Comment (andrew) [inline labelling.md]: Needs a space here between * and `wontfix`

**Comment by andrew** [comment]
Very nice 👏  We could link up each label across the org as well: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Alibrariesio+label%3A%22help+wanted%22

File: labelling.md (hunk 4 above)
Comment (BenJam) [inline labelling.md]: certainly.

**Comment by BenJam** [comment]
@andrew feedback addressed. I've added a note to add zeroday rather than a more generic security address so that maintainers, deployers etc can channel them appropriately.

**Pull request merged**